tests/ctst/features/azureArchive.feature

```diff
@@ -0,0 +1,129 @@
+        # |               Suspended |        100 |
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Archive 0 byte objects
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" that "exists"
+    And an object "obj2" that "exists"
+    And an object "obj3" that "exists"
+    Then the storage class of object "obj1" must become "e2e-azure-archive"
+    And the storage class of object "obj2" must become "e2e-azure-archive"
+    And the storage class of object "obj3" must become "e2e-azure-archive"
+
+    Examples:
+        | versioningConfiguration |
+        |           Non versioned |
+        |               Versioned |
+        # |               Suspended |
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Respect maximum number of objects per archived Tar
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And an object "obj1" of size <objectSize> bytes
+    And an object "obj2" of size <objectSize> bytes
+    And an object "obj3" of size <objectSize> bytes
+    And an object "obj4" of size <objectSize> bytes
+    And an object "obj5" of size <objectSize> bytes
+    And an object "obj6" of size <objectSize> bytes
+    Then the storage class of object "obj1" must become "e2e-azure-archive"
+    And the storage class of object "obj2" must become "e2e-azure-archive"
+    And the storage class of object "obj3" must become "e2e-azure-archive"
+    And the storage class of object "obj4" must become "e2e-azure-archive"
+    And the storage class of object "obj5" must become "e2e-azure-archive"
+    And the storage class of object "obj6" must become "e2e-azure-archive"
+    And manifest and tar containing object "obj1" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj2" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj3" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj4" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj5" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+    And manifest and tar containing object "obj6" should exist
+    And manifest containing object "" should contain <packObjectCount> objects
+
+    Examples:
+        | versioningConfiguration | objectSize | packObjectCount |
+        |           Non versioned |          1 |               3 |
+        |               Versioned |          1 |               3 |
+        # |               Suspended |          1 |               3 |
```

Comment:
should we have another config for "_non-versioned object in a versioned bucket_" ?

Reply:
i think we should wait for the follow up test ticket before adding this, as that case will change the semantics of the tests, we might want to make it it's own scenario

Reply:
ok, please add this to the followup ticket

Reply:
done